addons/op-badge/style.css

```diff
@@ -0,0 +1,19 @@
+.sa-original-poster {
+  display: inline-block;
+  background-color: var(--darkWww-button, #4d97ff);
```

Comment:
Could this be a different color in the forums when 2.0 → 3.0 is disabled - maybe `var(--darkWww-button-scratchr2PostHeader)`?

The font size should probably be set explicitly instead of relying on the default for `<small>`.

Reply:
> Could this be a different color in the forums when 2.0 → 3.0 is disabled - maybe `var(--darkWww-button-scratchr2PostHeader)`?

@mxmou Could you clarify? Did you mean "when dark mode is disabled" instead of "when 2.0 → 3.0 is disabled"?

Reply:
This shade of blue (`#4d97ff`) is a part of the Scratch 3.0 color scheme. It isn't used anywhere on scratchr2 pages unless 2.0 → 3.0 is enabled.

Reply:
So what color/CSS variable should we use in the forums?

Reply:
Probably `--darkWww-button-scratchr2PostHeader`.

Reply:
And what would be the fallback if website dark mode is disabled? 🤔

Reply:
`#28a5da`